tests/models/test_baseoperator.py

```diff
@@ -241,7 +243,46 @@ def test_default_resources(self):
         task = DummyOperator(task_id="default-resources")
         self.assertIsNone(task.resources)
 
+    # noinspection PyUnresolvedReferences
```

Comment:
What is the Unresolved Reference here?

Reply:
It's the qty field. The type hints are missing for Resource type clases and Intellij derives them from __init__ (names of __init__ parameters are the same even though they are different type long <> Resource. I will remove it and create an issue to fix type hints in the Resource classes.

Reply:
https://issues.apache.org/jira/browse/AIRFLOW-6406